docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+NeMo fundamentals
+=================
+
+On this page we will go a little bit deeper on how NeMo works, to give you a good foundation for using NeMo for your :ref:`desired usecase <where_next>`.
+
+NeMo Models
+-----------
+
+A NeMo "model" includes all of the below components wrapped into a singular, cohesive unit:
+
+* neural network architecture,
+
+* dataset & data loaders,
+
+* preprocessing of input data & postprocessing of model outputs,
+
+* loss function, optimizer & schedulers,
+
+* any other supporting infrastructure: tokenizers, language model configuration, data augmentation etc.
+
+NeMo models are based on PyTorch. Many of their components are subclasses of ``torch.nn.Module``. NeMo models use PyTorch Lightning (PTL) for training, thus reducing the amount of boilerplate code needed.
+
+NeMo models are also designed to be easily configurable; often this is done with YAML files. Below we show simplified examples of a NeMo model defined in pseudocode, and a config defined in YAML. We highlight the lines where the Python config parameter is read from the YAML file.
```

Comment:
remove comma.

NeMo models are also designed to be easily configurable; often this is done with YAML files. Below we show simplified examples of a NeMo model defined in pseudocode and a config defined in YAML. We highlight the lines where the Python config parameter is read from the YAML file.